### ISSUE TITLE
Documentation Suggestions

### ISSUE THREAD

**Issue by glenne** [domschl/RaspberryNtpServer#4]
Great tutorial!  I just got mine up and running.  Here's a few suggestions:

- Typo - Several references (4) to systemctl ... chronyd.  This should be just 'chrony', not 'chronyd'
- Typo - 'install gpsd via your **packet** manager.' should be 'install gpsd via your **package** manager.'
- Suggest including an example on how to install the first time an install is needed - 'sudo apt-get install gpsd'
- When adding allow 192.168/16 to chronyd.conf, one can omit the IP address and it will allow connections from any connected interface.
- A simple test from another computer can test connectivity.  On a mac, ```sntp <ip address>```, or verbose ```sntp -d <ip address>``` and it will show PPS0 being used.
- Documentation says 'If reception is ok, a led ("FIX" or "PPS") should start blinking on your GPS module.' On my Adafruit module, it blinks 1x/second if not fix and once every 10 seconds when there is a fix.

**Comment by Bob-VK6ZGN** [comment]
Yes Great tutorial! I just got mine up and running. 
Also using  Adafruit module and Rpi3 64bib Rasberian OS   All good
with one exception    Not sure what I am doing wrong But for SATS I just get [ -- ] when I have + 20 connected

any thoughts

**Comment by domschl** [comment]
@glenne: thanks for the list, will work on that during the next days
@Bob-VK6ZGN : probably something going wrong with the `gpsd-py3` dependency. Maybe open a separate issue for this, and we can try to track this down?

**Comment by glenne** [comment]
It would also be a bonus to explain what the time offset fields mean on the LCD.  e.g. NTP: and SATS

**Comment by domschl** [comment]
@glenne : I've added most of the suggestions:
some remarks:
- on some distris it is actually service 'chronyd', so I added both.
- `allow` without address (and especially `cmdallow` without) can lead to unexpected [results](https://github.com/domschl/RaspberryNtpServer#further-optimizations-in-chronyconf)

Thanks for the input!

I leave this open for further discussion. Feel free to close, if completed for now.
